docs/source/services/timelock_service/migration.rst

```diff
@@ -13,12 +13,37 @@ Otherwise, this can lead to serious data corruption.
 
 Automated Migration
 -------------------
+
 .. warning::
     Automated migrations are only implemented for Cassandra.
 
 1. If you are using the Casssandra key value service and have added the :ref:`Timelock client configuration <timelock-client-configuration>`, then starting/re-starting the service will automatically migrate the service.
 2. If you are using any other KVS, please follow the instructions at :ref:`manual-timelock-migration`.
 
+Verification
+~~~~~~~~~~~~
+
+It may be useful to verify that an automatic migration was carried out successfully. In addition to simply performing
+smoke tests of your clients and checking that they still work, there are a few other ways of checking that the
+migration occurred correctly.
+
+1. **Request Logs**: Find the TimeLock leader, and search its request logs for calls to the ``fastForwardTimestamp``
+   endpoint. These should be called with a ``currentTimestamp`` query parameter, which should be the bound in the
+   embedded timestamp bound store before the migration.
+2. **Key-Value Service table state**: Check the state of the key-value service. For Cassandra, you should expect
+   to see two rows: ``oldTs`` (the backed up value of the timestamp bound) and
+   ``ts`` (a bogus one byte array indicating that the timestamp table has been invalidated). The ``oldTs`` value
+   should match the ``currentTimestamp`` values in the request logs.
+
+.. code-block:: none
+
+   cqlsh> USE keyspace;
+   cqlsh:keyspace> SELECT * FROM "_timestamp";
+
+    key    | column1      | column2 | value
+   --------+--------------+---------+---------------------
+    0x7473 | 0x6f6c645473 |      -1 | 0x0001020304050607
```

Comment:
Is this the actual timestamp value, or is this the hex form of the value? It might be worth including the steps to confirm that this number is equal to what you will find in your logs.